### PR TITLE
add support for mips64el and mips64

### PR DESCRIPTION
**Pull request by wanghuaiqing2010**
Signed-off-by: wanghuaiqing <wanghuaiqing@loongson.cn>

### CODE REVIEW

**Comment by wanghuaiqing2010** [comment]
@ tonistiigi
I can not see any log. 
 could you tell me why ci/build (mainline-all) (pull_request) failed ?

File: docker-bake.hcl [hunk 1]
```diff
@@ -21,7 +21,7 @@ target "binaries" {
 }
 
 target "all-arch" {
-  platforms = ["linux/amd64", "linux/arm64", "linux/arm", "linux/ppc64le", "linux/s390x", "linux/riscv64", "linux/386"]
+  platforms = ["linux/amd64", "linux/arm64", "linux/arm", "linux/ppc64le", "linux/s390x", "linux/riscv64", "linux/386", "linux/mips64el"]
```
Comment (tonistiigi) [inline docker-bake.hcl:24]: Error is
```
#10 [linux/mips64el internal] load metadata for docker.io/library/base-mips64el:latest
#10 sha256:ac77c6a20c5733238141da74d482e6cbc88691987e8b76744ff534d439afa53b
#10 ERROR: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```
Don't add mips in this file as releasing image for mips is more complicated due to missing base images etc. and not that important. It's good enough just to build mips emulator for other platforms.

Comment (wanghuaiqing2010) [reply docker-bake.hcl:24]: @tonistiigi Thank you very much

**Review by tonistiigi — APPROVED** [review]

**Comment by tonistiigi** [comment]
@wanghuaiqing2010 Now that https://github.com/moby/buildkit/pull/1959 is merged please update `archutil` vendoring in here so the output also shows if mips is supported.

**Comment by wanghuaiqing2010** [comment]
@ tonistiigi sure, it is my pleasure.

**Pull request merged**